packages/react-docsite-components/src/components/Markdown/Markdown.tsx

```diff
@@ -33,93 +33,96 @@ const MarkdownBase: React.FunctionComponent<IMarkdownProps> = props => {
 
   return (
     <div className={classNames.root}>
-      <MarkdownToJsx {...getMarkdownProps(classNames.subComponentStyles, props)}>{children}</MarkdownToJsx>
+      <MarkdownComponent options={getMarkdownOptions(classNames.subComponentStyles, props)}>
+        {children}
+      </MarkdownComponent>
     </div>
   );
 };
 MarkdownBase.displayName = 'Markdown';
 
-function getMarkdownProps(subComponentStyles: IMarkdownSubComponentStyles, props: IMarkdownProps): MarkdownToJsxProps {
+function getMarkdownOptions(
+  subComponentStyles: IMarkdownSubComponentStyles,
+  props: IMarkdownProps,
+): MarkdownToJSX.Options {
   return {
-    options: {
```

Comment:
This is best viewed with ignore whitespace changes--it's just switching to return the former `options` prop directly instead of wrapped, since for some reason the latest markdown-to-jsx doesn't explicitly export a props type for its Markdown component (but it does have a type for the options).